public/js/test/mochitest/browser_dbg_interrupts.js

```diff
@@ -0,0 +1,89 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/**
+ * Test resuming from button and keyboard shortcuts.
+ */
+
+const TAB_URL = EXAMPLE_URL + "doc-script-switching-01.html";
+const sourceURLs = [
+  "code-script-switching-01.js",
+  "code-script-switching-02.js"
+];
+const resumeKey = "VK_F8";
+
+add_task(function*() {
+  const dbg = yield initDebugger(TAB_URL, ...sourceURLs);
+  const { selectors: { getBreakpoints, getBreakpoint }, getState } = dbg;
+  const sources = [];
+  for (let url of sourceURLs) {
+    sources.push(findSource(dbg, url));
+  }
+
+  yield ensureAddBreakpoint(dbg, getState, getBreakpoint, sources[0].id, 5);
+  yield ensureAddBreakpoint(dbg, getState, getBreakpoint, sources[1].id, 6);
+  yield ensureAddBreakpoint(dbg, getState, getBreakpoint, sources[1].id, 7);
+  is(getBreakpoints(getState()).size, 3, "Three breakpoint exists");
+
+  yield testResumeButton(dbg);
```

Comment:
I think it would be better to expand some of this out into a couple lines

```js
invokeInTab("firstCall");
yield waitForPaused(dbg);
```

also, i think passing `.button.active` as a full selector is fine